Description: There is something wrong when using reqwest/rustls-tls
Forwarded: not-needed
Last-Update: 2025-02-18
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -309 +309 @@
-    "rustls",
+    "native-tls", #"rustls",
@@ -319,2 +319,2 @@
-rustls = [
-    "reqwest/rustls-tls",
+#rustls = [
+#    "reqwest/rustls-tls",
@@ -322,3 +322,3 @@
-    "reqwest/rustls-tls-native-roots",
-    "dep:rustls",
-]
+#    "reqwest/rustls-tls-native-roots",
+#    "dep:rustls",
+#]
